### PR TITLE
feat: logistic regression classifier service

### PR DESCRIPTION
**Pull request by jamesthatcher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ML-python-sklearn-lrc-cpu-service
+
+Quickstart project for executing an Wine classifier using the SciKit-Learn framework on a CPU.
+
+Running pip install requirements.txt and then python `app.py` will start the app on localhost where the user can send GET requests to perform inference.
```
Comment (mxdillon) [inline README.md]: can you put the `pip`  and whole `python` commands in the apostrophe things so md renders as code?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+scikit-learn==0.21.3
+future==0.17.1
+protobuf==3.7.1
+falcon==1.4.1
+gunicorn==19.9.0
+numpy==1.16.2
+pytest==5.3.2
+onnxruntime==1.1.0
```
Comment (mxdillon) [inline requirements.txt]: can you put in alphabetical order for ease of navigation

File: test/test_app.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def test_classification_request(client):
+    """Expected classification for Wine sample should be returned"""
+    client.app.add_route('/wine/{index:int(min=0)}', Wine())
+
+    result = client.simulate_get('/wine/1')
+    assert result.status == "200 OK", "The service test will fail until a trained model has been approved"
+    assert all(k in result.json for k in (
+        "index", "predicted_label", "predicted")), "The service test will fail until a trained model has been approved"
```
Comment (mxdillon) [inline test/test_app.py]: what are index, predicted_label, predicted and where are they set?

**Comment by mxdillon** [comment]
I think minor update to `requirements.txt` is outstanding

File: test/test_app.py (hunk 3 above)
Comment (jamesthatcher) [reply test/test_app.py:23]: These are set in `service.py` as a response to a given GET request.

**Pull request merged**